tools/go.mod

```diff
@@ -12,3 +12,4 @@ require (
 	github.com/gvallee/validation_tool v1.3.1
 	gonum.org/v1/plot v0.9.0
 )
+
```

Comment:
That should not be required, these files are managed mainly through `go` commands